### PR TITLE
13.2.0

### PR DESCRIPTION
**Pull request by GHarte**

### CODE REVIEW

File: docs/android/customisation/settingsMenu.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
```
Comment (MarkMan2407) [inline docs/android/customisation/settingsMenu.md:2]: Can the copy be consistent for Android and iOS as far as possible for the description of functionality

File: docs/android/customisation/settingsMenu.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Settings Menu
+{: .no_toc }
+
+Customization of the landing and search screen toolbar settings menu button. 
+
+---
+
+## Setting an icon 
+We have provided three icons to chose from(cog, hamburger and user): <br />
```
Comment (MarkMan2407) [inline docs/android/customisation/settingsMenu.md:18]: Inconsistent copy between two platforms

File: docs/android/customisation/settingsMenu.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Settings Menu
+parent: Customisation
+grand_parent: Android Integration
+nav_order: 8
+permalink: /docs/android/customisation/settings/
+---
+
+# Settings Menu
```
Comment (MarkMan2407) [inline docs/android/customisation/settingsMenu.md:10]: Title and sentence below - can they be the same for both platforms?

File: docs/ios/customisation/settings.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import CarTrawlerSDK
+let context = CTContext(clientID: "your client ID", flow: .standAlone)
+context.countryCode = "IE" 
+context.currencyCode = "EUR" 
+context.languageCode = "EN"
+
+context.settingsIconType = .cog // or .hamburger, or .user
+```   
+
+{: .note}
```
Comment (MarkMan2407) [inline docs/ios/customisation/settings.md:33]: Is there a reason this note might not be relevant for Android. If so, might be good add it there also.

**Review by firminofranciele007 — APPROVED** [review]

**Pull request merged**